CustomerData/PrivateData.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Metrilo\Analytics\CustomerData;
+
+use Magento\Customer\CustomerData\SectionSourceInterface;
+
+class PrivateData implements SectionSourceInterface
+{
+    public function __construct(
+        \Metrilo\Analytics\Helper\SessionEvents $sessionEvents
+    ) {
+        $this->sessionEvents = $sessionEvents;
+    }
+    
+    public function getSectionData()
+    {
+        $events = $this->sessionEvents->getSessionEvents();
+        return ['metrilo_events' => $events];
```

Comment:
is this key needed at all?

Reply:
getSectionData() method should return key => val array according to best practice examples provided in the documentation.

Reply:
link

Reply:
https://devdocs.magento.com/guides/v2.3/extension-dev-guide/cache/page-caching/private-content.html

https://github.com/magento/magento2/blob/2.3/app/code/Magento/Catalog/CustomerData/CompareProducts.php#L36-L45

Reply:
> The public method getSectionData must return an array with data for private block.
isn't `$events` an array?

Reply:
$events variable is array, the key part is passed to knockoutjs component as a method. Technically i believe this can be done without key, but this will complicate the js logic in the component where not only the metrilo events is present as data but also the section id witch is updated after data change.

Reply:
ok, fair point.
Do we need the `metrilo_` part?
`metrilosection().metrilo_events` seems to duplicate `metrilo`

Reply:
Will remove the metrilo_ part from the section key since it's unnecessary.